### ISSUE TITLE
Z Blending with other pipelines

### ISSUE THREAD

**Issue by wmiller848** [StarArawn/bevy_tiled#7]
I noticed an issue when adding a bevy sprite sheet. There is something odd occurring where the frame buffer is not blending the components of different passes.

In the screen shot you'll notice that the two layers in the tile map are blended correctly but the sprite is ignored.

I've tried changing bevy upstream to use `depth_compare: CompareFunction::LessEqual` but that didn't seem to have any effect. It looks like the frame buffer is being overridden on the tile map draw pass.

https://github.com/StarArawn/bevy_tiled/compare/master...wmiller848:sprite_blend_issue

<img width="1392" alt="depth_issue" src="https://user-images.githubusercontent.com/2379758/91063430-d6f56f00-e5e2-11ea-8aa6-0c0dd69cb7ac.png">

**Comment by wmiller848** [comment]
So this is an artifact of `position.z = layer_id;` in the shader. If you apply the z-translation via a translation when building the chunk it fixes the issue.

**Comment by StarArawn** [comment]
Yeah that sounds like the correct approach to take. Ideally we should have a transform per layer. I think whenever I get around to it I can refactor the entities/components to look something like this: `World Entity > Layer Entity > Chunk Entity`. Chunk entities should have their own transform probably too.
